README.md

```diff
@@ -173,12 +173,13 @@ go test -v ./...
 
 You can set the following environment variables to change the behavior of knuu:
 
-| Environment Variable       | Description                               | Possible Values         | Default |
-|----------------------------|-------------------------------------------|-------------------------|---------|
-| `KNUU_TIMEOUT`             | The timeout for the tests.                | Any valid duration       | `60m`    |
-| `KNUU_NAMESPACE`           | The namespace where the instances will be created. | Any valid namespace name | `test`  |
-| `KNUU_BUILDER`             | The builder to use for building images.   | `docker`, `kubernetes`  | `docker` |
-| `DEBUG_LEVEL`              | The debug level.                          | `debug`, `info`, `warn`, `error` | `info`  |
+| Environment Variable        | Description                                        | Possible Values                  | Default  |
+|-----------------------------|----------------------------------------------------|----------------------------------|----------|
+| `KNUU_TIMEOUT`              | The timeout for the tests.                         | Any valid duration               | `60m`    |
+| `KNUU_NAMESPACE`            | The namespace where the instances will be created. | Any valid namespace name         | `test`   |
```

Comment:
My intuition would be if `KNUU_DEDICATED_NAMESPACE` is `true` and `KNUU_NAMESPACE` is set, it should use `KNUU_NAMESPACE` as the namespace. This is up to you. Either way, please explain this in the docs.